src/fluree/db/full_text.clj

```diff
@@ -3,30 +3,28 @@
             [fluree.db.dbproto :as dbproto]
             [fluree.db.flake :as flake]
             [fluree.db.full-text.block-registry :as block-registry]
-            [clojure.java.io :as io]
             [clojure.string :as str]
             [clojure.walk :refer [keywordize-keys]]
             [clucie.analysis :as lucene-analysis]
             [clucie.core :as lucene]
             [clucie.store :as lucene-store])
-  (:import fluree.db.flake.Flake
-           java.io.Closeable
-           java.io.File
-           org.apache.lucene.analysis.Analyzer
-           org.apache.lucene.analysis.en.EnglishAnalyzer
-           org.apache.lucene.analysis.cn.smart.SmartChineseAnalyzer
-           org.apache.lucene.analysis.hi.HindiAnalyzer
-           org.apache.lucene.analysis.es.SpanishAnalyzer
-           org.apache.lucene.analysis.ar.ArabicAnalyzer
-           org.apache.lucene.analysis.id.IndonesianAnalyzer
-           org.apache.lucene.analysis.ru.RussianAnalyzer
-           org.apache.lucene.analysis.bn.BengaliAnalyzer
-           org.apache.lucene.analysis.br.BrazilianAnalyzer
-           org.apache.lucene.analysis.fr.FrenchAnalyzer
-           org.apache.lucene.index.IndexWriter
-           org.apache.lucene.index.IndexReader
-           org.apache.lucene.index.IndexNotFoundException
-           org.apache.lucene.store.Directory))
+  (:import (fluree.db.flake Flake)
```

Comment:
Yes, two actually:

1. consistency (internally and with the more common community practice / likely linter defaults)
2. it's more concise when you need to import more than one symbol like `(foo.bar Baz Qux)`

Reply:
cool. this is fine by me, but my practice was to only use the paren form when i wanted to import multiple classes from the namespace (or whatever that java equivalent term is). my reason was also for concision. you save 3 whole characters! (and i think the non-paren form is a little easier to read)

i'm good with changing my practice to go with the flow

Reply:
Yeah I don't think it matters too terribly much either way. I read somewhere years ago that the `(class.path Symbol)` format was the Right Way™ b/c it mirrored the `:require` format more closely and it uses parens (instead of square brackets) b/c the first element is different / significant kind of like function call position is in normal lisp code. It also allows editors to more easily add / remove new / vestigial imports automatically for you as you change the code, which I kind of like. What happened here IIRC was that my editor added the `Flake` import in this format and the difference stuck out to me so I changed the others.